docs/3.api/3.utils/define-nuxt-component.md

```diff
@@ -36,6 +36,10 @@ export default defineNuxtComponent({
 </script>
 ```
 
+::warning
+Data from `useAsyncData` is only set once in the `setup` hook of your component. This means `refreshNuxtData` will not work with `asyncData`. If you need to re-run `useAsyncData`, you can change it with a `method` of the Options API.
```

Comment:
```suggestion
Data from `asyncData` is only set once in the `setup` hook of your component. This means `refreshNuxtData` will not work with `asyncData`. If you need to re-run `useAsyncData`, you can change it with a `method` of the Options API.
```

Reply:
does it work to call `refreshNuxtData` later on (ie not in setup function, but in an event handler)? if so, I think we can clarify this wording slightly